crates/graphql/src/schema/context.rs

```diff
@@ -81,6 +81,7 @@ pub struct AppContext {
     pub nft_attributes_loader: Loader<PublicKey<Nft>, Vec<NftAttribute>>,
     pub nft_by_mint_loader: Loader<PublicKey<TokenMint>, Option<Nft>>,
     pub nft_collection_loader: Loader<PublicKey<Nft>, Option<Collection>>,
+    pub generic_collection_loader: Loader<String, Option<Collection>>,
```

Comment:
```suggestion
    pub generic_collection_loader: Loader<CollectionIdentifier, Option<Collection>>,
```

@ray-kast attempted to to `pub CollectionIndentifier(String)` but got a good amount of trait errors from Loader. We are pressing on but a spot that could use a refactor.